github/actions_artifacts_test.go

```diff
@@ -0,0 +1,243 @@
+// Copyright 2020 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"reflect"
+	"testing"
+)
+
+func TestActionsService_ListWorkflowRunArtifacts(t *testing.T) {
+	client, mux, _, teardown := setup()
+	defer teardown()
+
+	mux.HandleFunc("/repos/o/r/actions/runs/1/artifacts", func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, "GET")
+		testFormValues(t, r, values{"page": "2"})
+		fmt.Fprint(w,
+			`{
+				"total_count":1, 
+				"artifacts":[{"id":1}]
+			}`,
+		)
+	})
+
+	opt := &ListOptions{Page: 2}
```

Comment:
When you get a chance, could you please change `opt` to `opts` to be consistent with #1417?
(Here and anywhere else you find it in this PR, please.)

Reply:
Fixed.